api/pacman-api-asset/src/main/java/com/tmobile/pacman/api/asset/repository/PacmanRedshiftRepository.java

```diff
@@ -39,13 +39,13 @@
 @Repository
 public class PacmanRedshiftRepository {
 
-    @Value("${redshift.url}")
+    @Value("${redshift.url:}")
```

Comment:
May I know the reason for this change?

Reply:
These are not part from our PR. We have not done any code changes in Pacman-api-asset.